.devops/nix/package.nix

```diff
@@ -157,6 +163,8 @@ effectiveStdenv.mkDerivation (
       substituteInPlace ./*.py --replace "/usr/bin/env python" "${llama-python}/bin/python"
     '';
 
+    __noChroot = effectiveStdenv.isDarwin && useMetalKit;
```

Comment:
Add a comment with a permalink to the github discussion and the brief summary of the xcode/metal situation

Reply:
A link to the PR you found earlier, making `metal` mandatory, would also be nice